checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -116,17 +121,36 @@ function StartSchedulerRunner {
         [string]$ConfigFilePath
     )
 
-    $config = [Config]::ParseConfigFile($ConfigFilePath)
-    $commandSpec = CreateSchedulerExecCommand $ConfigFilePath $config.RCCConfig
-
-    if (-not (Test-Path $config.LogDirectory)) {
-        New-Item -Path $config.LogDirectory -ItemType Directory
+    try {
+        $config = [Config]::ParseConfigFile($ConfigFilePath)
+    }
+    catch {
+        # What should the path of the log files be?
+        # Here we still don't have access to the defined path from the config file
+        WriteLogAndException -Message $_.Exception.Message -LogPath "" -ExceptionLogPath ""
+        throw
     }
 
     $selfLogPath = Join-Path $config.LogDirectory "scheduler_runner.log"
+    $exceptionLogPath = Join-Path $config.LogDirectory "scheduler_runner"
```

Comment:
Here we are defining the path of the agent output-like log file, but I am not sure what it should be, so I just chose this.

Reply:
Not sure what you mean by "agent output-like log file". The config object has a `ResultsDirectory` field. In this directory, we can write agent output.

Reply:
Makes sense. I will adjust it.